docs/changing_plugins_install_dir.md

```diff
@@ -1,6 +1,8 @@
 # Changing plugins install dir
 
-By default, TPM installs plugins to `~/.tmux/plugins/`.
+By default, TPM installs plugins in a subfolder named `plugins/` inside
+`$XDG_CONFIG_HOME/tmux/` if a `tmux.conf` file was found at that location, or
+inside `~/,tmux/` otherwise.
```

Comment:
Typo comma instead of dot on the line above.

Reply:
Fixed.